apps/web/src/components/Composer/Actions/Attachment.tsx

```diff
@@ -64,23 +64,44 @@ const Attachment: FC<Props> = ({ attachments, setAttachments }) => {
     return false;
   };
 
+  const isImageType = (files: any) => {
+    for (const file of files) {
+      if (ALLOWED_IMAGE_TYPES.includes(file.type)) {
+        return true;
+      }
+    }
```

Comment:
When we select multiple files and `isImageType` is called with multiple `files` it will return true if first image file type is found 😅 is this expected? Because the false is returned only when none of the files are image. but true is returned if any one of the file type is image?

Reply:
Thank you for pointing out my mistake, I didn't consider the case where the first selected file is not an image when selecting multiple files. I will carefully check my code logic and test it before submitting it next time. 🙏

Reply:
Yea NP lets fix it rest looks good to me ✅